### ISSUE TITLE
Minimal role for MySQL / MariaDB

### ISSUE THREAD

**Issue by alanorth** [ilri/rmg-ansible-public#1]
I'm not so much interested in managing all teh things, but rather being able to manage the config files for MySQL server. Ideally the role would at a minimum:
- allow for a host to choose MySQL or MariaDB
- allow for managing my.cnf or probably `/etc/mysql/conf.d/ilri.conf`

I think we should investigate some common best practices for MySQL / MariaDB servers, based on guides like these where Oracle and friends are re-examining the defaults in their products:
- https://blogs.oracle.com/supportingmysql/entry/server_defaults_changes_in_mysql
- https://www.percona.com/blog/2014/01/28/10-mysql-settings-to-tune-after-installation/

**Comment by oguya** [comment]
We can have one big role - `roles/db` - which allows you to choose which DBMS(MySQL,PostgreSQL or MariaDB) you'd like to deploy.
Then, you can have a variable - `preferred_dbms` - which will contain the name of the dbms you'd like to deploy.

**Comment by alanorth** [comment]
Yes, I think this is similar to how I did XMPP and how we talked of doing web server:

https://github.com/alanorth/ansible-personal/blob/prosody/roles/xmpp/defaults/main.yml#L6
